apps/server/src/modules/deletion/repo/deletion-request.repo.spec.ts

```diff
@@ -146,24 +146,24 @@ describe(DeletionRequestRepo.name, () => {
 				em.clear();
 
 				const expectedArray = [
-					{
-						id: deletionRequestEntity4.id,
-						targetRefDomain: deletionRequestEntity4.targetRefDomain,
-						deleteAfter: deletionRequestEntity4.deleteAfter,
-						targetRefId: deletionRequestEntity4.targetRefId,
-						status: deletionRequestEntity4.status,
-						createdAt: deletionRequestEntity4.createdAt,
-						updatedAt: deletionRequestEntity4.updatedAt,
-					},
-					{
-						id: deletionRequestEntity3.id,
-						targetRefDomain: deletionRequestEntity3.targetRefDomain,
-						deleteAfter: deletionRequestEntity3.deleteAfter,
-						targetRefId: deletionRequestEntity3.targetRefId,
-						status: deletionRequestEntity3.status,
-						createdAt: deletionRequestEntity3.createdAt,
-						updatedAt: deletionRequestEntity3.updatedAt,
-					},
+					// {
+					// 	id: deletionRequestEntity4.id,
+					// 	targetRefDomain: deletionRequestEntity4.targetRefDomain,
+					// 	deleteAfter: deletionRequestEntity4.deleteAfter,
+					// 	targetRefId: deletionRequestEntity4.targetRefId,
+					// 	status: deletionRequestEntity4.status,
+					// 	createdAt: deletionRequestEntity4.createdAt,
+					// 	updatedAt: deletionRequestEntity4.updatedAt,
+					// },
+					// {
+					// 	id: deletionRequestEntity3.id,
+					// 	targetRefDomain: deletionRequestEntity3.targetRefDomain,
+					// 	deleteAfter: deletionRequestEntity3.deleteAfter,
+					// 	targetRefId: deletionRequestEntity3.targetRefId,
+					// 	status: deletionRequestEntity3.status,
+					// 	createdAt: deletionRequestEntity3.createdAt,
+					// 	updatedAt: deletionRequestEntity3.updatedAt,
+					// },
```

Comment:
Looks like some leftover, please delete these commented out lines if they're not needed in the end.

Reply:
done